nautobot/extras/tests/example_jobs/test_atomic_transaction.py

```diff
@@ -22,9 +27,10 @@ def run(self, fail=False):
             Status.objects.create(name="Test database atomic rollback 1")
             if fail:
                 raise SimulatedError("simulated failure")
-        except Exception:
-            self.log_failure("Job failed, all database changes have been rolled back.")
-        self.log_success("Job succeeded.")
+        except Exception as err:
+            logger.error("Job failed, all database changes have been rolled back.")
+            raise err
```

Comment:
Maybe here and below:

```suggestion
        except Exception:
            logger.error("Job failed, all database changes have been rolled back.")
            raise
```